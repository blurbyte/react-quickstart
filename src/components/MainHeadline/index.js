import styled from 'styled-components';

const MainHeadline = styled.h1`
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 5rem;
  letter-spacing: -0.2rem;
  color: #ff4326;
  border-top: 2px solid #908e89;
  padding-top: 1rem;
  margin-bottom: 2.4rem;
  line-height: 1.2;
`;

export default MainHeadline;
