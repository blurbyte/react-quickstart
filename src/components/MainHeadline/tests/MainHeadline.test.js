import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import MainHeadline from '../index';

describe('<MainHeadline />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <MainHeadline />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
