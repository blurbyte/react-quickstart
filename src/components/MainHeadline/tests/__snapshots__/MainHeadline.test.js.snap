// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainHeadline /> should render correctly 1`] = `
.jSBCfo {
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 5rem;
  letter-spacing: -0.2rem;
  color: #ff4326;
  border-top: 2px solid #908e89;
  padding-top: 1rem;
  margin-bottom: 2.4rem;
  line-height: 1.2;
}

<h1
  className="sc-bdVaJa jSBCfo"
/>
`;
